Sources/KSCrashRecording/include/KSCrashMonitorType.h

```diff
@@ -81,21 +84,37 @@ typedef enum
     KSCrashMonitorTypeUserReported       | \
     KSCrashMonitorTypeSystem             | \
     KSCrashMonitorTypeApplicationState   | \
-    KSCrashMonitorTypeZombie               \
+    KSCrashMonitorTypeZombie             | \
+    KSCrashMonitorTypeMemoryTermination    \
+)
+
+#define KSCrashMonitorTypeFatal            \
+(                                          \
+    KSCrashMonitorTypeMachException      | \
+    KSCrashMonitorTypeSignal             | \
+    KSCrashMonitorTypeCPPException       | \
+    KSCrashMonitorTypeNSException        | \
+    KSCrashMonitorTypeMainThreadDeadlock   \
 )
 
 #define KSCrashMonitorTypeExperimental     \
 (                                          \
     KSCrashMonitorTypeMainThreadDeadlock   \
 )
 
+/*
 #define KSCrashMonitorTypeDebuggerUnsafe   \
 (                                          \
     KSCrashMonitorTypeMachException      | \
     KSCrashMonitorTypeSignal             | \
     KSCrashMonitorTypeCPPException       | \
     KSCrashMonitorTypeNSException          \
 )
+*/
+#define KSCrashMonitorTypeDebuggerUnsafe   \
+(                                          \
+    KSCrashMonitorTypeMachException     \
```

Comment:
Mach exception are really the only kind of monitor that will really mess with the debugger.

Reply:
Looks plausible because most `SIGNAL`s can be disabled in the settings